### DISCUSSION TITLE
OAuth 2.0

### DISCUSSION THREAD

**Discussion by yosiasz** [sindresorhus/got#1595] [Q&A] [answered]
Greetings

amazing module and thanks for it. Does it have support for OAuth 2.0 . Meaning can I use got with OAuth 2.0 

Thanks!

**Comment by szmarczak** [comment, marked as answer]
OAuth is a standard. Got can work with anything, you have just to implement it. There is no native support for OAuth. You'd have to use a package like this: https://github.com/sindresorhus/got#oauth

**Reply by yosiasz** [reply]
thanks much! Will find oAuth2 module since the api I am hitting requires oAuth2
